starfish/intensity_table/intensity_table.py

```diff
@@ -416,7 +420,7 @@ def to_expression_matrix(self, regions: Optional[regional.many]=None) -> Express
         """
 
         # create the 2-d counts matrix
-        grouped = self.to_features_dataframe().groupby(['cell_id', 'target'])
+        grouped = self.to_features_dataframe().groupby(['features', 'target'])
         counts = grouped.count().iloc[:, 0].unstack().fillna(0)
 
         if regions:
```

Comment:
are there any pipelines where we know the cell_ids of the spots but don't have region (or segmentation masks) data?

Reply:
ping

Reply:
I've updated the ISS vignette to include the finding of cell ids for some of the spots. That data can be used as an example to create testing data, or saved and used directly here.